### PR TITLE
Add claim provider and governance connector

### PR DESCRIPTION
**Pull request by sadilchamishka**

### CODE REVIEW

File: components/org.wso2.carbon.identity.organization.management.claim.provider/src/main/java/org/wso2/carbon/identity/organization/management/claim/provider/OrganizationClaimProvider.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                String organizationName = getOrganizationManager().getOrganizationNameById(organizationId);
+                additionalClaims.put(ORGANIZATION_ID_ATTRIBUTE, organizationId);
+                additionalClaims.put(ORGANIZATION_NAME_ATTRIBUTE, organizationName);
+            }
+        } catch (OrganizationManagementClientException e) {
+            if (ERROR_CODE_ORGANIZATION_NOT_FOUND_FOR_TENANT.getCode().equals(e.getErrorCode())) {
+                return additionalClaims;
+            } else {
+                throw new IdentityOAuth2Exception("Error while resolving organization id.", e);
+            }
```
Comment (ravinduramesh) [inline components/org.wso2.carbon.identity.organization.management.claim.provider/src/main/java/org/wso2/carbon/identity/organization/management/claim/provider/OrganizationClaimProvider.java:104]:
```suggestion
            if (!ERROR_CODE_ORGANIZATION_NOT_FOUND_FOR_TENANT.getCode().equals(e.getErrorCode())) {
                throw new IdentityOAuth2Exception("Error while resolving organization id.", e);
            } 
            return additionalClaims;
```

Comment (sadilchamishka) [reply components/org.wso2.carbon.identity.organization.management.claim.provider/src/main/java/org/wso2/carbon/identity/organization/management/claim/provider/OrganizationClaimProvider.java]: Resolved the suggession in this manner - https://github.com/wso2-extensions/identity-organization-management/pull/223/commits/a63c34e0bdada46c4bfb8b8e2509e644066c4e14

File: components/org.wso2.carbon.identity.organization.management.claim.provider/src/main/java/org/wso2/carbon/identity/organization/management/claim/provider/internal/OrganizationClaimProviderServiceComponent.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+            service = OrganizationManager.class,
+            cardinality = ReferenceCardinality.MANDATORY,
+            policy = ReferencePolicy.DYNAMIC,
+            unbind = "unsetOrganizationManager"
+    )
+    protected void setOrganizationManager(OrganizationManager organizationManager) {
+
+        if (LOG.isDebugEnabled()) {
+            LOG.debug("Setting the organization management service.");
+        }
```
Comment (AnuradhaSK) [inline components/org.wso2.carbon.identity.organization.management.claim.provider/src/main/java/org/wso2/carbon/identity/organization/management/claim/provider/internal/OrganizationClaimProviderServiceComponent.java:72]: Let's remove if check.
Also move the debug log after the action 
Change the log as a completed action `Set the organization management service.`

Check the other places as well

Comment (sadilchamishka) [reply components/org.wso2.carbon.identity.organization.management.claim.provider/src/main/java/org/wso2/carbon/identity/organization/management/claim/provider/internal/OrganizationClaimProviderServiceComponent.java]: Ack. Fixed by - https://github.com/wso2-extensions/identity-organization-management/pull/223/commits/ce6255751687edc48f0de001f94b8d72964fb7df

**Review by AnuradhaSK — APPROVED** [review]

**Pull request merged**